perceval/backends/core/gitlab.py

```diff
@@ -373,6 +379,7 @@ class GitLabClient(HttpClient, RateLimitHandler):
          before raising a RetryError exception
     :param archive: an archive to store/read fetched data
     :param from_archive: it tells whether to write/read the archive
+    :param is_oauth_token: True if the token is OAuth and not private/personal
```

Comment:
Depending on the outcome of the previous comment, this param should be moved accordingly.